{"name": "test_login[data2]", "status": "passed", "attachments": [{"name": "log", "source": "9544cdb5-d51b-461c-bb0f-bd89c549ca1a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'case_id': 'login_case_002', 'case_title': '用户输入正确的用户名、\\n错误的密码，点击登录', 'test_data': {'username': 'alive', 'password': 1234567}, 'exp': {'by': 'xpath', 'locater': \"//*[@id='common-prompt']/div/p\", 'exp_value': '密码错误'}}"}], "start": 1663422391242, "stop": 1663422397246, "uuid": "48b7ea72-cce4-498d-a540-eaed232ac53c", "historyId": "178f5884c0b5708ea915e2eac7a5a413", "testCaseId": "c6db3f0583de5245fe1a62b1d735a86c", "fullName": "run.TestLogin#test_login", "labels": [{"name": "suite", "value": "run"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-5FKMFO0H"}, {"name": "thread", "value": "25936-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "run"}]}